libs/balances-and-allowances/src/constants/bff-balances-swr-config.ts

```diff
@@ -3,12 +3,52 @@ import { SWRConfiguration } from 'swr'
 
 import { BASIC_MULTICALL_SWR_CONFIG } from '../consts'
 
+let focusLostTimestamp: number | null = null
+let listenersInitialized = false
+const FOCUS_HIDDEN_DILAY = ms`10s`
+
+function initializeFocusListeners(): void {
+  if (listenersInitialized || typeof document === 'undefined') {
```

Comment:
If we call `initializeFocusListeners` just in the file root, we won't need listenersInitialized flag, no?